### DISCUSSION TITLE
what are Iranians fighting for?

### DISCUSSION THREAD

**Discussion by heidiidbs** [WeAreMahsaAmini/Core#48] [Ideas]
Add a section in the website for "what are Iranians fighting for?" to add, gather and translate as many "برای..." tweets as possible. This will remind everyone why we are doing this, as well as reflecting the world about all the pains we have endured for 43 years. This will be a strong message I believe.
@syavash @farnoodma I have already started this project by my own and have more detailed ideas on this. Please let me know how I should add content or start the project here if you would like.

**Comment by syavash** [comment]
Amazing! @keyvankhademi is working on the website project. He can help.

In the meantime, if you already have a list, you can open a PR and add a new folder and index.md file [here](https://github.com/WeAreMahsaAmini/Homepage/tree/main/content/articles/)

If it's easier for you to paste it here that will work as well. We'll put it in.

Come by to Discord to chat about your ideas. I'm usually there on a voice channel after 7pm ET.
